### PR TITLE
fix minor typo in examples/instantiate/docs_example

### PR DESCRIPTION
**Pull request by Jasha10**
This PR fixes a minor typo in `fix minor typo in examples/instantiate/docs_example/my_app.py`.

### CODE REVIEW

**Review by jieru-hu — APPROVED** [review]
lgtm. Thanks!

Looks like the CI failed due to some infra issue, I've kicked them off again. Will merge once all green.

**Comment by omry** [comment]
Please wait until the 1.1 release is done before merging anything.

**Review by omry — APPROVED** [review]

**Pull request merged**
